src/v2/Apps/Partner/Components/PartnerArtists/PartnerArtistsCarousel/PartnerArtistsCarouselItem.tsx

```diff
@@ -0,0 +1,103 @@
+import React from "react"
+import styled from "styled-components"
+import {
+  Box,
+  ResponsiveBox,
+  Image,
+  EntityHeader,
+  Flex,
+  space,
+} from "@artsy/palette"
+import { ContextModule } from "@artsy/cohesion"
+import { createFragmentContainer, graphql } from "react-relay"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { FollowArtistButtonFragmentContainer } from "v2/Components/FollowButton/FollowArtistButton"
+import { PartnerArtistsCarouselItem_artist } from "v2/__generated__/PartnerArtistsCarouselItem_artist.graphql"
+
+export const ResponsiveImage = styled(ResponsiveBox).attrs({
+  aspectWidth: 3,
+  aspectHeight: 2,
+  maxWidth: "100%",
+  maxHeight: "100%",
+  width: "100%",
+  bg: "black10",
+})<any>``
```

Comment:
Why the any?

Reply:
I getting a type error. I tried cut already set attrs using Omit, but it didn't help

![image](https://user-images.githubusercontent.com/79979820/115730105-b1498100-a38e-11eb-91d2-ce2948391e80.png)

Reply:
Ah, I see; no worries then